stream-chat-android/src/main/java/com/getstream/sdk/chat/CaptureMediaContract.kt

```diff
@@ -46,14 +46,20 @@ public class CaptureMediaContract : ActivityResultContract<Unit, File>() {
     private fun createFileName(prefix: String, extension: String) =
         "${prefix}_${dateFormat.format(Date().time)}.$extension"
 
+    private fun getFileProviderAuthority(context: Context): String {
+        val compName = ComponentName(context, StreamFileProvider::class.java.name)
```

Comment:
`StreamFileProvider` class should have a different name after obfuscation. Won't it break anything?

Reply:
The `StreamFileProvider::class` reference should still point to the class correctly, even if it does get obfuscated. Since the provider is an entry point for the app, I'd expect it not to be obfuscated in the first place though.